streamwebs_frontend/streamwebs/static/gis/js/OLMapWidget.js

```diff
@@ -0,0 +1,386 @@
+/*global OpenLayers*/
```

Comment:
Sequel to the above comment ^^: Originally I didn't read Travis output carefully so I assumed that this file was the one that was needed for the checks to pass...